### ISSUE TITLE
'smarthome:automation listRules' causes an exception

### ISSUE THREAD

**Issue by openhab-5iver** [eclipse/smarthome#5976]
OH 2.4 snapshot 1319 (ESH 0.10.0.201807241640) (this does not occur on snapshot 1304)

I am using the Experimental Rules Engine (JSR223). Running this in Karaf results in an exception...
```
openhab> smarthome:automation listRules
An unexpected error occurred during execution.
```
The openhab.log shows this...
```
2018-07-28 00:24:22.471 [ERROR] [rg.eclipse.smarthome.io.console.ConsoleInterpreter] - An error occurred while executing the console command.
java.lang.NullPointerException: null
        at org.eclipse.smarthome.automation.internal.commands.Printer.printRules(Printer.java:115) [210:org.eclipse.smarthome.automation.commands:0.10.0.201807241640]
        at org.eclipse.smarthome.automation.internal.commands.AutomationCommandList.listRules(AutomationCommandList.java:178) [210:org.eclipse.smarthome.automation.commands:0.10.0.201807241640]
        at org.eclipse.smarthome.automation.internal.commands.AutomationCommandList.execute(AutomationCommandList.java:85) [210:org.eclipse.smarthome.automation.commands:0.10.0.201807241640]
        at org.eclipse.smarthome.automation.internal.commands.AutomationCommands.executeCommand(AutomationCommands.java:517) [210:org.eclipse.smarthome.automation.commands:0.10.0.201807241640]
        at org.eclipse.smarthome.automation.internal.commands.AutomationCommandsPluggable.execute(AutomationCommandsPluggable.java:184) [210:org.eclipse.smarthome.automation.commands:0.10.0.201807241640]
        at org.eclipse.smarthome.io.console.ConsoleInterpreter.execute(ConsoleInterpreter.java:56) [111:org.eclipse.smarthome.io.console:0.10.0.201807241640]
        at org.eclipse.smarthome.io.console.karaf.internal.CommandWrapper.execute(CommandWrapper.java:79) [112:org.eclipse.smarthome.io.console.karaf:0.10.0.201807241640]
        at org.apache.karaf.shell.impl.console.osgi.secured.SecuredCommand.execute(SecuredCommand.java:68) [12:org.apache.karaf.shell.core:4.1.5]
        at org.apache.karaf.shell.impl.console.osgi.secured.SecuredCommand.execute(SecuredCommand.java:86) [12:org.apache.karaf.shell.core:4.1.5]
        at org.apache.felix.gogo.runtime.Closure.executeCmd(Closure.java:571) [12:org.apache.karaf.shell.core:4.1.5]
        at org.apache.felix.gogo.runtime.Closure.executeStatement(Closure.java:497) [12:org.apache.karaf.shell.core:4.1.5]
        at org.apache.felix.gogo.runtime.Closure.execute(Closure.java:386) [12:org.apache.karaf.shell.core:4.1.5]
        at org.apache.felix.gogo.runtime.Pipe.doCall(Pipe.java:417) [12:org.apache.karaf.shell.core:4.1.5]
        at org.apache.felix.gogo.runtime.Pipe.call(Pipe.java:229) [12:org.apache.karaf.shell.core:4.1.5]
        at org.apache.felix.gogo.runtime.Pipe.call(Pipe.java:59) [12:org.apache.karaf.shell.core:4.1.5]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:?]
        at java.lang.Thread.run(Thread.java:748) [?:?]
```

**Comment by openhab-5iver** [comment]
This still occurs in OH snapshot 1321.

**Comment by openhab-5iver** [comment]
This appears to be resolved in OH snapshot 1325 (ESH 0.10.0.201808011124).
